src/python/pants/backend/scala/dependency_inference/scala_parser_test.py

```diff
@@ -165,3 +181,13 @@ class NestedClass {
             ),
         }
     )
+
+    assert analysis.consumed_symbols_by_scope == FrozenDict(
+        {
+            "org.pantsbuild.example.OuterClass.NestedObject": FrozenOrderedSet(["String"]),
+            "org.pantsbuild.example.Functions": FrozenOrderedSet(
+                ["bar", "foo", "OuterObject.NestedVal", "+", "Unit"]
```

Comment:
`bar` and `foo` are false positives here, right? I think that the comment you had in the Java consumed types extraction was helpful for tracking that.

Reply:
In this case, they aren't because it is a test and those were the names I chose.

Reply:
Mm... I guess also that between Java and Scala we moved from "consumed types" to "consumed symbols"... and these are definitely symbols.

Reply:
The `"+"` comes from the `Term.ApplyInfix` AST node which is for infix expressions. Probably worth filtering those out, but let's get some experience with using this for inference first. and adjust accordingly.